cli.go

```diff
@@ -12,6 +12,7 @@
 //			 Usage: "say a greeting",
 //			 Action: func(c *cli.Context) error {
 //				 println("Greetings")
+//         return nil
```

Comment:
:nitpick: indentation

I also find it amusing that this example uses `println`